src/analyses/ai.cpp

```diff
@@ -237,33 +240,37 @@ bool ai_baset::fixedpoint(
 
   // Put the first location in the working set
   if(!goto_program.empty())
-    put_in_working_set(
-      working_set,
-      goto_program.instructions.begin());
+  {
+    locationt first = goto_program.instructions.begin();
+    put_in_working_set(working_set, start_history(first));
+    /// TODO : should only start history once?
```

Comment:
Fix me

Reply:
Fixed in the next patch set; to get one start and function calls you have to de-recursion the AI.